README.md

```diff
@@ -0,0 +1 @@
+Testing project for learning
```

Comment:
[Severity: Low] The README file lacks a newline at the end of the file.
Add a newline at the end of the file to adhere to common coding standards.
Rationale: Many tools and editors expect a newline at the end of files, which can prevent issues in version control and improve compatibility with various systems.
Reference: The Unix convention for text files states that a newline character should be present at the end of the file.